### ISSUE TITLE
Cannot find module 'solito/link' or its corresponding type declarations.

### ISSUE THREAD

**Issue by ankit-tailor** [nandorojo/solito#150]
After upgrading to version `1.0.0` getting this error.
`Cannot find module 'solito/link' or its corresponding type declarations.`

### Steps to reproduce

1. yarn add solito@latest
2. Getting above error 🔝 

### Code
```
import { Link as SolitoLink } from 'solito/link'
import React from 'react'
import { Center, Button, Box } from 'native-base'

export function HomeScreen() {
  return (
    <Center
      flex={1}
      _dark={{ bg: 'blueGray.900' }}
      _light={{ bg: 'blueGray.50' }}
    >
      <Box mt="6">
        <SolitoLink href="/">
          <Button pointerEvents="none" variant="outline" colorScheme="coolGray">
            Hello world
          </Button>
        </SolitoLink>
      </Box>
    </Center>
  )
}
```

### Aditional Information
- In `node_modules` solito contains `build/index.js` file and `package.json` other directories(link, moti, router) are not there.

**Comment by nandorojo** [comment]
i think i know what’s up. does 0.0.29 work?

**Comment by ankit-tailor** [comment]
Yes, it's working.

**Comment by nandorojo** [comment]
Fixed in `1.0.10`. Thanks for offering!
